packages/css/src/logo/logo.scss

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+.amsterdam-logo {
+  display: inline-block;
+}
+
+.amsterdam-logo__image {
+  height: 40px;
+  vertical-align: middle;
+  width: auto;
```

Comment:
Is this necessary? `width: auto` is the default, no?

Reply:
Yes it's the default, I left it there because it was previously a pixel value which prevented layout shift. We should probably go for a width and height attribute in the svg instead of viewBox.